mem.go

```diff
@@ -1,18 +1,25 @@
 package qfs
 
-import "fmt"
+// MemFSStore is the minimum interface for creating a MemFS
+type MemFSStore interface {
```

Comment:
This interface definition isn't specific to *Mem*FSStores, since it only deals with string -> File getting. Could we use a more generic name like FSStore?

Reply:
oooh v good point. on it